app/src/main/java/org/oppia/app/home/WelcomeViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.home
+
+import androidx.databinding.ObservableField
+import androidx.lifecycle.ViewModel
+
+/** [ViewModel] for user app usage history. */
```

Comment:
Actually sorry but I missed this comment in my last PR review. We will need to update this to
`/** [ViewModel] for welcome text in home screen. */`

Reply:
Done :)